### DISCUSSION TITLE
Lisp Assembly at 38c3 This Month

### DISCUSSION THREAD

**Discussion by jgarte** [jank-lang/jank#144] [Issue Triage] [answered]
# Announcement

Hi jank community,

I thought the following announcement by [Fabio Natali](https://social.coop/@fnat) of an event coming up in Hamburg would be interesting to the jank community so I am paraphrasing the original post for you below.

https://events.ccc.de/congress/2024/hub/en/assembly/lisp/

## TLDR

[Fabio Natali](https://fabionatali.com/) has registered a Lisp assembly at 38c3. 
Fabio is currently reaching out to various Lisp-related communities to
see if anyone is also attending 38c3 and might be interested in joining
the assembly.

## 38c3

[38c3](https://events.ccc.de/congress/2024/hub/en/assembly/lisp/) is the 38th edition of the Chaos Communication Congress or [CCC]( https://en.wikipedia.org/wiki/Chaos_Communication_Congress),
the annual hacker conference happening in Hamburg, Germany from 27th to
30th December. Roughly speaking, a Congress assembly is a space that is
made available for a self-organized group, to collaborate, organize
workshops, talks, etc. around a specific project or area of interest.

The [38c3 Lisp assembly](https://events.ccc.de/congress/2024/hub/en/assembly/lisp/) is meant as a meeting point for Lisp
aficionados of all levels as well as for the passer-by who's curious and
wants to know more. The hope is that this event will attract friends from a variety of
Lisp dialects (e.g. Chicken, Clojure, Common Lisp, Emacs Lisp, Fennel,
Guile, Racket, jank) and Lisp-based projects (e.g. Emacs and Guix).

### Activities 

Fabio would like to prepare a small number of activities, such as:

- micro talks,
- pair-programming sessions,
- bug-reporting/bug-fixing/patch-review sessions.

## Questions

If you're interested or have any questions, you can:

- write to ~fabionatali/lisp-at-38c3@lists.sr.ht ([public list](https://lists.sr.ht/~fabionatali/lisp-at-38c3/)), or
- reply to this discussion, or
- reach out on the Fediverse at @fnat@social.coop, or
- reach out on IRC at fnat at libera.chat.

**Comment by jeaye** [comment, marked as answer]
I apologize for never responding to this. It was lost in my email and since we weren't using Discussions much at the time, I never saw it again.

Thank you for reaching out here. Please continue to keep the jank community updated for future events. I want to hear about them and I'm sure others do, too!
